ksqldb-engine/src/main/java/io/confluent/ksql/function/udaf/BaseComparableKudaf.java

```diff
@@ -44,7 +45,7 @@ public BaseNumberKudaf(
         outputSchema,
         Collections.singletonList(
             new ParameterInfo(
-                "number",
+                "comparable",
```

Comment:
This word still makes some noise. 

This is what `describe function max;` displays when `number` is used:
```
ksql> describe function max;

Name        : MAX
Author      : Confluent
Type        : AGGREGATE
Jar         : internal
Variations  : 

	Variation   : MAX(number INT)
	Returns     : INT
	Description : Computes the maximum integer value for a key.
	number      : the value to aggregate

	Variation   : MAX(number BIGINT)
	Returns     : BIGINT
	Description : Computes the maximum long value for a key.
	number      : the value to aggregate
        
       ...
```

It now displays `comparable`:
```
ksql> describe function max;

Name        : MAX
Author      : Confluent
Type        : AGGREGATE
Jar         : internal
Variations  : 

	Variation   : MAX(comparable INT)
	Returns     : INT
	Description : Computes the maximum INTEGER value for a key.
	comparable  : the value to aggregate

	Variation   : MAX(comparable BIGINT)
	Returns     : BIGINT
	Description : Computes the maximum BIGINT value for a key.
	comparable  : the value to aggregate

       ...
```

The `comparable` should be something else that reflects what the parameter is used in those functions, i.e. `number`, `date`, `time`, etc.

Reply:
Would "value" be a suitable replacement for "comparable"?  Worst case, things could be reworked so that each of the constructors could pass in a different word there.  E.g., "number" for Int,BigInt,etc, and date/time for those new types.